### ISSUE TITLE
Rough receive audio on MacOS 10.15.6

### ISSUE THREAD

**Issue by jimcam** [nostar/dudestar#81]
Cloned and compiled the main branch today on MacOS 10.15.6   All good except for the receive audio has repeated ( ~1 second), very short (~100ms)  dropouts and just as frequent clicks.   

Thank you for putting your efforts into this package.  Well done!

**Comment by nostar** [comment]
Im using 10.15.7 on a modest MacBook Pro 13.  All modes receive perfectly smooth when the source server provides the data in a timely matter.  There a many reasons why this can happen including unstable internet connection on your side or at the host, overloaded machine, etc.  There is no data that this is an issue with dudestar.

**Comment by jimcam** [comment]
I was on a remote DSL connection to the network with poor performance.   I have changed locations to a fios based internet provider and with that it sounds much smoother and certainly better.     And then out of curiosity I switched to my cell phone hotspot, not as smooth as fios but better than DSL.

Is there enough buffering of packets or would adding some buffering of packets help to smooth out the audio?  Just curious....
